itest/lnd_zero_conf_test.go

```diff
@@ -177,8 +177,11 @@ func testZeroConfChannelOpen(ht *lntest.HarnessTest) {
 	payReq := eve.RPC.DecodePayReq(eveInvoiceResp.PaymentRequest)
 	require.Len(ht, payReq.RouteHints, 0)
 
-	// Make sure Dave is aware of this channel and send the payment.
-	ht.AssertTopologyChannelOpen(dave, fundingPoint3)
+	// Make sure Alice which is not a direct peer will see this channnel
+	// as well after it is confirmed. A public zeroconf channel is only
+	// announced to the broader network after six confirmations.
+	ht.AssertTopologyChannelOpen(ht.Alice, fundingPoint3)
```

Comment:
don't think this change is necessary - if we wanna test relaying gossip we should test it in graph tests.

Reply:
Ok will remove